src/Component.php

```diff
@@ -203,7 +204,17 @@ private function handleIf( DOMNodeList $nodes, array $data ) {
 					$nodesToRemove[] = $node;
 				}
 
-				$previousIfCondition = $condition;
+				$previousIfCondition = $previousIfCondition || $condition;
+			} elseif ( $node->hasAttribute( 'v-else-if' ) ) {
+				$conditionString = $node->getAttribute( 'v-else-if' );
+				$node->removeAttribute( 'v-else-if' );
+				$condition = $this->app->evaluateExpression( $conditionString, $data );
+
+				if ( !$condition ) {
```

Comment:
I think this conditional needs to be changed like this to avoid all true `else-if`s being rendered:

```php
if ( $previousIfCondition || !$condition ) {
```